api/src/paths/dwc/submission/{submissionId}/normalize.ts

```diff
@@ -57,7 +57,8 @@ POST.apiDoc = {
                 type: 'integer',
                 minimum: 1
               }
-            }
+            },
+            nullable: true
```

Comment:
Why nullable? Since this is the 200 response, we should be able to control the response and always return consistently. We could decide that returning the submission_status_id isn't helpful, but in that case we should just remove the entire content section and just return `200`.

Reply:
Technically there is nothing wrong with having different possible return values, just feels odd to me for w/e reason